Source/Demagnetization.H

```diff
@@ -7,13 +7,18 @@
 #include <cufft.h>
 #endif
 
+#include <heffte.h>
+
 using namespace amrex;
 
 class Demagnetization {
 
     BoxArray ba_large;
     DistributionMapping dm_large;
     Geometry geom_large;
+
+    Box domain_large;
+    RealBox real_box_large;
```

Comment:
Delete these lines, in the .cpp file we declare these as temporary variables that go out of scope after ::define() so we don't need to keep these around.